aria/tasks_graph.py

```diff
@@ -58,9 +58,13 @@ def tasks(self):
         for _, data in self.graph.nodes_iter(data=True):
             yield data['task']
 
-    def tasks_in_order(self, reverse=False):
+    def tasks_in_order(self, reverse=False, dependencies=False):
```

Comment:
tasks_in_order give us tasks in order
create for this another method
def tasks_tree(reverse=False):
    for task in self.tasks_in_order(reverse):
        yield task, self.task_dependencies(task)